rc/base/spell.kak

```diff
@@ -9,6 +9,10 @@ def -params ..1 spell -docstring "Check spelling of the current buffer with aspe
         printf %s\\n "set buffer spell_tmp_file ${file}"
     }
     %sh{
+        _echo() {
```

Comment:
Expanding a variable with arbitrary content is unsafe with `echo`, it's not just about minus signs (backslashes matter as well), and since I don't want to have to think about whether a variable can contain those characters constantly, I use `printf`. Just like when I have to use a function with a hardcoded format string more than a couple times, I abstract it in `_echo` so that I don't litter the code with a parameter that is irrelevant (we only want to print a stupid string).